### ISSUE TITLE
Determine Path Forward for Zendesk Transition

### ISSUE THREAD

**Issue by melanienleopold** [18F/tts-tech-portfolio#1243]
## Background information

Zendesk is FedRAMPed, but teams are currently not using the FedRAMPed version of Zendesk. All of the teams on Zendesk need to transition to the new version, but it is a large cost increase for all programs (4x the cost).

## Implementation Steps
- [ ] Get an extension on the current ATO for Zendesk while we transition
- [ ] Move programs under the pcard limit to Zendesk version at the time of their renewal
- [x] Meet with cloud.gov and login.gov to determine next steps for an acquisition

## Acceptance criteria

- [ ] Consensus from all teams using Zendesk and Security on the timeline for continuing/ending Zendesk use

**Comment by melanienleopold** [comment]
Waiting on Security's response to the ATO extension request. Met with the program teams on 2/18. Moving forward with contacting vendors on Schedules to see what is available and requesting a GSA ServiceNow demo.

**Comment by melanienleopold** [comment]
Waiting on FITARA and Legal review for the Search.gov account renewal and waiting to hear back from FedRAMP if they want to renew their account.

**Comment by melanienleopold** [comment]
From Nate: 
The ZenDesk ATO has been extended until 15 May, but that extension was granted so that GSA could discontinue the use of ZenDesk. If you are planning to use the Enterprise version of ZenDesk we'll need to complete a CRM package.

From Tamara:
If you want to use the FedRAMP version of ZenDesk, I just need a technical contact to work on the SSP (needs to be able to answer to the controls, supply network diagrams, etc).  Please let me know whom from your team will be doing that.

Can we push back to see if we can have until August like asked in the letter? There is little chance we can completely roll over by May with having to do an acquisition. Who on the team can be the technical contact?

**Comment by melanienleopold** [comment]
Additionally, there are many terms and conditions in Zendesk's agreement that GSA cannot agree to. We are meeting with Zendesk tomorrow to determine a path forward.

**Comment by melanienleopold** [comment]
Blocked by [#979](https://github.com/18F/tts-tech-portfolio-private/issues/979)

**Comment by melanienleopold** [comment]
Zendesk did not accept the government's rider. GSA legal is going back and forth with Zendesk legal.

**Comment by melanienleopold** [comment]
Per Zendesk: "I'm working with our product team in an effort to modify your subscription to remove the ability to add 3rd party applications but this process will take 6-8 weeks to complete.  If we can successfully remove this feature then it's possible our legal team will agree to the section in question.  I should have a better answer for you in the next few weeks but I appreciate your patience here."

**Comment by afeld** [comment]
Closing in favor of https://github.com/18F/tts-tech-portfolio/issues/1300 and https://github.com/18F/tts-tech-portfolio/issues/978.
